include/buildpack.bash

```diff
@@ -176,3 +176,54 @@ buildpack-execute() {
 	mv $build_path/* $app_path
 	shopt -u dotglob nullglob
 }
+
+buildpack-test() {
+	declare desc="Run an application tests using installed buildpacks"
+	ensure-paths
+	[[ "$USER" ]] || randomize-unprivileged
+	buildpack-setup > /dev/null
+
+	if [[ -n "$BUILDPACK_URL" ]]; then
```

Comment:
There is some code duplication here. I'd rather see some of this broken out into a function and get re-used by this and anywhere else we handle buildpack detection.